src/gdcdictionary/schemas/follow_up.yaml

```diff
@@ -1747,44 +1788,6 @@ properties:
         $ref:
           - "_terms_enum.yaml#/not_reported/common"
 
-  hpv_positive_type:
```

Comment:
do we usually delete the property before the migration is done? (https://gdc-ctds.atlassian.net/browse/DAT-3528)?

Reply:
Actually ignore what I said above, migration has been completed 👍